### ISSUE TITLE
模型运行速度3000毫秒，同样模型tflite300毫秒

### ISSUE THREAD

**Issue by Whitman1984** [alibaba/MNN#1156]
模型为mobilenetv1为基础的Unet结构，在高通骁龙625上，四线程cpu运行速度为3000毫秒左右，同样的模型转成tflite运行速度大概在300毫秒，
def MOBILENETV1_Modified_UNet(weights=None, input_tensor=None):
    mobilenet = keras.applications.mobilenet.MobileNet(include_top=False, weights='imagenet', input_tensor=input_tensor)
    mobilenet.summary()
    source = mobilenet.get_layer('conv_pw_11_relu').output
    x = MaxPooling2D(3, strides=1, padding='same', name='conv_pw_11_relu_pool')(source)
    x = Conv2D(512, kernel_size=3, padding='same', dilation_rate=6)(x)
    x = Conv2D(512, kernel_size=1)(x)
    #x = Interpolate(target_shape=_collect_input_shape(source)[1:3], name='resize_1')(x)
    x = concatenate([x, source])
    x = up_conv(x, [256, 128])
    source = mobilenet.get_layer('conv_pw_5_relu').output
    x = Lambda(Interpolate)([x, source])
    # x = Interpolate(target_shape=_collect_input_shape(source)[1:3], name='resize_2')(x)
    x = concatenate([x, source])
    x = up_conv(x, [128, 64])
    source = mobilenet.get_layer('conv_pw_3_relu').output
    x = Lambda(Interpolate)([x, source])
    # x = Interpolate(target_shape=_collect_input_shape(source)[1:3], name='resize_3')(x)
    x = concatenate([x, source])
    x = up_conv(x, [64, 32])
    source = mobilenet.get_layer('conv_pw_1_relu').output
    x = Lambda(Interpolate)([x, source])
    # x = Interpolate(target_shape=_collect_input_shape(source)[1:3], name='resize_4')(x)
    x = concatenate([x, source])
    feature = up_conv(x, [32, 32])
    x = conv_cls(feature, 2)
    region_score = Lambda(out_layer1)(x)
    affinity_score = Lambda(out_layer2)(x)
    return region_score, affinity_score
这是我的模型结构
用opencl速度大概在210毫秒，
用vulkan报错如下
![image](https://user-images.githubusercontent.com/55910777/94534095-94265700-0272-11eb-85ce-734ede312842.png)
[craft_model_mobilenetv1_modified_median_599.zip](https://github.com/alibaba/MNN/files/5303273/craft_model_mobilenetv1_modified_median_599.zip)

**Comment by jxt1234** [comment]
mnn 什么时候的版本呢?

**Comment by jxt1234** [comment]
问题已知，这个模型有个卷积层 batch 很大而 width / height 较小，我们看一下怎么修复吧

**Comment by Whitman1984** [comment]
0.2.1.5git

**Comment by jxt1234** [comment]
目前 Convolution 3x1 部分还待优化，1x1 已经优化了，目前单线程 600 ms 左右，后面再完善吧
